### PR TITLE
Feature/delete single data points

### PR DESCRIPTION
**Pull request by Mark-Krensel**
Please review my code for single data point deletion.
Styling will be addressed in a coming user story.

### CODE REVIEW

File: pages/api/Days/index.js
Comment (philmetscher) [inline pages/api/Days/index.js]: remove console.log

**Review by philmetscher — CHANGES_REQUESTED** [review]
for me it looks fine. But i haven't started with DB structuring yet. So let's have a second review

File: pages/api/Days/index.js
Comment (Mark-Krensel) [reply pages/api/Days/index.js]: done

File: components/Card.js
Comment (Mengzhen1992) [inline components/Card.js:72]: Hi Mark, a small suggestion: you have used DeleteSingleButton three times, maybe it would be better when one uses it as a component so that the card page would have less code. The other looks good to me.

File: components/Card.js
Comment (ThomasF85) [inline components/Card.js]: It's a little cleaner if you call the method with one argument only `handleDelete(id)` and give the method default values for the second and third parameters.

**Review by ThomasF85 — APPROVED** [review]

File: components/Card.js
Comment (Mark-Krensel) [reply components/Card.js]: done

**Pull request merged**
